src/main/kotlin/uk/gov/justice/digital/hmpps/nomisprisonerapi/core/SplashScreenResource.kt

```diff
@@ -62,20 +62,83 @@ class SplashScreenResource(private val service: SplashScreenService) {
   fun getScreenConditions(
     @Schema(description = "The name of the screen (module)", example = "OIDINCRS")
     @PathVariable moduleName: String,
-  ): SplashScreenDto = service.getSplashScreens(moduleName)
+  ): SplashScreenDto = service.getSplashScreen(moduleName)
+
+  @PreAuthorize("hasRole('ROLE_NOMIS_SCREEN_ACCESS')")
+  @GetMapping("/splash-screens/{moduleName}/blocked")
+  @Operation(
+    summary = "Retrieve a list of blocked prison ids for the screen",
+    description = "Retrieves a list of blocked prison ids for the screen (module) name or **ALL* if all prisons, or an empty list if there are none blocked. Requires role NOMIS_SCREEN_ACCESS",
```

Comment:
V. minor type on `**ALL*`

Reply:
oopsie - will add in - I am doing another PR so will include in that